upt_macports/templates/base.Portfile

```diff
@@ -4,19 +4,21 @@ PortSystem          1.0
 {% block portgroup %}
 {% endblock %}
 
-# Portfile generated/maintained using the Universal Packaging Tool (upt)
+## Portfile generated/maintained using the Universal Packaging Tool (upt)
+## Please check out the commented statements and check the guide for more information:
+##     https://guide.macports.org/chunked/project.contributing.html
 {% block nameversion %}
 {% endblock %}
 
-# Please adapt the lines with a "TODO" statement and check the guide for more information:
-# https://guide.macports.org/#reference.keywords
-
-# categories-append # TODO
+# categories-append 
 platforms           darwin
-# TODO: uncomment the following line if no architecture-dependent files are installed, otherwise remove
+## uncomment the following line if no architecture-dependent files are installed, otherwise remove
 # supported_archs     noarch
 license             {{ pkg.licenses }}
-# maintainers         {example.org:julesverne @GitHub-handle} openmaintainer # TODO
+## If you would like to keep maintaining this port in the future as described in
+## please add your contacts (e-mail, github handle), else keep `nomaintainer`.
```

Comment:
So if we are keeping both the options, should we do something like keep the `nomaintainer` line uncommented and the other as option?
so it would be like this: 

```
## If you would like to keep maintaining this port in the future as described in
## https://guide.macports.org/chunked/project.contributing.html then
## please add your contacts (e-mail, github handle), otherwise keep the 'nomaintainer' line
# maintainers         {example.org:julesverne @github-handle} openmaintainer
maintainers         nomaintainer
```

Reply:
yes, that's indeed a better idea - otherwise it will not generate a "valid" Portfile and ```port lint``` will complain about it. (As a small nitpick, please attempt to keep the length of the comment line as equal as possible).